domain-story-modeler-thesis/docker-compose.yml

```diff
@@ -8,17 +8,4 @@ services:
     ports:
       - "9080:80"
     environment:
-      - BACKEND_URL=http://localhost:8888/
-
-  # rabbit:
-  #   container_name: "api-rabbit"
-  #   image: rabbitmq:3-management
-  #   ports:
-  #     - "15672:15672"
-  #     - "5672:5672"
-  #   healthcheck:
-  #     test: rabbitmq-diagnostics check_port_connectivity
-  #     interval: 2s
-  #     timeout: 5s
-  #     retries: 20
-  #     start_period: 5s
+      - VITE_BACKEND_URL=PROVIDE_YOUR_BACKEND_URL
```

Comment:
Instead of PROVIDE_YOUR_BACKEND_URL consider a comment since the env variable is not empty and the container will start although a URL is not provided by the user